projects/1.20-fabric/better-archeology/betterarcheology/lang/zh_cn.json

```
@@ -0,0 +1,94 @@
+{
+  "itemGroup.betterarcheology": "更好的考古学",
+
+  "block.betterarcheology.rotten_log": "腐烂原木",
+  "block.betterarcheology.rotten_planks": "腐烂木木板",
+  "block.betterarcheology.rotten_slab": "腐烂木台阶",
+  "block.betterarcheology.rotten_stairs": "腐烂木楼梯",
+  "block.betterarcheology.rotten_fence": "腐烂木栅栏",
+  "block.betterarcheology.rotten_fence_gate": "腐烂木栅栏门",
+  "block.betterarcheology.rotten_trapdoor": "腐烂木活板门",
+  "block.betterarcheology.rotten_door": "腐烂木门",
+  "block.betterarcheology.infested_mud_bricks": "疫病泥砖",
+  "block.betterarcheology.cracked_mud_bricks": "裂纹泥砖",
+  "block.betterarcheology.cracked_mud_brick_slab": "裂纹泥砖台阶",
+  "block.betterarcheology.cracked_mud_brick_stairs": "裂纹泥砖楼梯",
+  "block.betterarcheology.suspicious_red_sand": "可疑的红沙",
+  "block.betterarcheology.suspicious_dirt": "可疑的泥土",
+  "block.betterarcheology.fossiliferous_dirt": "含化石的泥土",
+  "block.betterarcheology.fossiliferous_dirt_tooltip": "也许其中有什么东西等待着发掘？",
+  "block.betterarcheology.villager_fossil": "村民化石",
+  "block.betterarcheology.villager_fossil_tooltip": "它变为化石的手臂或许可以展示什么东西……",

```

| **审查级别**    |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思  |

原文：**Its fossilized arms could act as a sort of display**

原文没有省略号

```suggestion
  "block.betterarcheology.villager_fossil_tooltip": "它变为化石的手臂或许可以拿去作展品",
```
```suggestion
  "block.betterarcheology.villager_fossil_tooltip": "它变为化石的手臂或许可以拿去展览",
```

---

译文不做改动。因为村民化石确实是可以充当物品展示架的。